simulations/vip-357/utils/cut-params-mainnet.json

```diff
@@ -0,0 +1,73 @@
+{
+  "cutParams": [
+    [
+      "0x4b093a3299F39615bA6b34B7897FDedCe7b83D63",
+      1,
+      [
+        "0xa76b3fda",
+        "0x929fe9a1",
+        "0xc2998238",
+        "0xede4edd0",
+        "0xb0772d0b",
+        "0xabfceffc",
+        "0x007e3dd2",
+        "0xc488847b",
+        "0xa78dc775",
+        "0xddbf54fd"
```

Comment:
I miss `"0xc5b4db55"` (`venusInitialIndex()`), that was added in the [VIP-181](https://app.venus.io/#/governance/proposal/181?chainId=56)